docker/Dockerfile.buster

```diff
@@ -0,0 +1,26 @@
+FROM python:3.7-buster AS builder
+
+RUN useradd --user-group --create-home zulip 
+# && apt-get update \
+# && apt-get install -y libffi-dev libxml2-dev libxslt1-dev 
```

Comment:
Can these lines be removed, since we don't run them?

Reply:
You are right. During development I've found that base image already contains these:
```
libffi-dev is already the newest version (3.2.1-9).
libxml2-dev is already the newest version (2.9.4+dfsg1-7+b3).
libxslt1-dev is already the newest version (1.1.32-2).
```
I'll update that

Reply:
To answer your question. Testing really isn't difficult. It may be just tedious to remove cached images for the clear run. It's slightly better to use Alpine as a baseline, because it is barebone and you have to be more explicit with each build step.

The idea to use volume to keep the config file come afterwards.

Reply:
I didn't initially explore Alpine as I didn't realize the build process was so separated from the build OS that this was possible - but that builds and runs for me with a few adjustments to the run command, as I noted elsewhere.